src/simulations/set_generator.py

```diff
@@ -51,6 +52,48 @@ def __iter__(self):
     for _ in range(self.num_sets):
       yield self.set_size
 
+def _choice_fast(n, m, random_state):
```

Comment:
Would it make sense to put this into its own file?  I think that we are going to have a number of different set generation methods, which will probably span multiple files.

Reply:
It could definitely make sense to put this into its own file. My original logic was that this was a niche function that was only going to be used in this file. Currently, I believe that set_generator.py is the only file that contains (multiple) set generation methods so for now I believe the scope of this function should be specific to set_generator.py. If it is guaranteed that we will start separating out the set generation methods into multiple files, let me know! I will definitely consider putting it in a different file if that is the case.

Reply:
@jgoodknight mentioned that there was a common folder as part of our package that I missed! This method definitely belongs there instead. Fixed in 38fadaa